core/src/main/java/cucumber/api/formatter/NiceAppendable.java

```diff
@@ -65,7 +65,7 @@ public NiceAppendable println(CharSequence csq) {
     public void close() {
         try {
             tryFlush();
-            if (out instanceof Closeable && out != System.out && out != System.err) {
+            if (out instanceof Closeable) {
```

Comment:
This is the responsibility of the caller who passes in System.out or System.error